### ISSUE TITLE
older messages are append bottom instead of top

### ISSUE THREAD

**Issue by shankaranand001** [twilio/twilio-conversations-demo-react#65]
while open the conversation details with messages, default load the 30 recent messages, while scroll up older messages came, but these older messages append to bottom instead of top. can you please help, thank you.

**Comment by kouloughli-hemza** [comment]
same problem , is there a way to fix this ?

**Comment by berkus** [comment]
Yeah, should change the way loaded messages are populated. This shall be easier with our new ChatLog components.

You can investigate [around here](https://github.com/twilio/twilio-conversations-demo-react/blob/main/src/components/message/MessagesBox.tsx#L101-L116) yourself, we will probably only get to this later.

**Comment by kouloughli-hemza** [comment]
> 

I will investigate that , Thank you

**Comment by agrukhal** [comment]
Closed as there was no follow-up. Should the problem persist, please, open a new issue
